Core/Core/API/APIRequestable.swift

```diff
@@ -154,23 +203,31 @@ extension APIRequestable {
     }
     public var isBodyFromURL: Bool { false }
 
+    public var useExtendedPercentEncoding: Bool { false }
+
     public func urlRequest(relativeTo baseURL: URL, accessToken: String?, actAsUserID: String?) throws -> URLRequest {
         guard var components = URLComponents(string: path) else { throw APIRequestableError.invalidPath(path) }
 
         if !path.hasPrefix("/") && components.host == nil {
             components.path = "/api/v1/" + components.path
         }
 
-        var queryItems = self.queryItems
+        var actAsUserQueryItem: [URLQueryItem] = []
```

Comment:
Nice renaming! We code change this to a let with an inline function initializer to reflect that this variable won't change.